config/yosbr/config/relicex.json

```diff
@@ -1,15 +1,15 @@
 {
   "chestsHaveLoot": true,
-  "chestsHaveRelicChance": 15,
+  "chestsHaveRelicChance": 28,
   "chestsHaveLesserOrbChance": 8,
-  "chestsHaveGreaterOrbChance": 2,
-  "chestsHaveTomeChance": 7,
+  "chestsHaveGreaterOrbChance": 1,
+  "chestsHaveTomeChance": 0,
   "dropsOnlyFromPlayerKills": true,
   "dragonDropsStone": false,
-  "mobsDropLootChance": 3,
+  "mobsDropLootChance": 4,
   "mobDropIsRelicChance": 50,
   "mobDropIsPotionChance": 0,
-  "mobDropIsLesserOrbChance": 50,
-  "mobDropIsGreaterOrbChance": 3,
-  "mobDropIsTomeChance": 1
-}
\ No newline at end of file
+  "mobDropIsLesserOrbChance": 35,
+  "mobDropIsGreaterOrbChance": 1,
+  "mobDropIsTomeChance": 0
```

Comment:
This is something that should be left up to a vote.
I like the change, seeing as there are other methods of getting tomes, but they are still rare enough that the community should decide.

Reply:
The point is to make tomes not drop at all from any source, if we're making levelling exponentially harder at higher levels we should not have the possibility of one getting essentially thousands of XP levels from a single mob drop.